### PR TITLE
Bug post page

### PR DESCRIPTION
**Pull request by knargocer**
Fixed the bugs in PostList and Post components

### CODE REVIEW

File: src/components/Post/index.jsx [hunk 1]
```diff
@@ -4,40 +4,27 @@ import { useTranslation } from "react-i18next";
...
     );
   }
-  return (
-    <iframe
-      title="load"
-      src="https://giphy.com/embed/xTk9ZvMnbIiIew7IpW"
-      width="480"
-      height="480"
-    />
-  );
```
Comment (Ammar-64) [inline src/components/Post/index.jsx:42]: Add else here

File: src/components/SuccessStories/index.jsx [hunk 2]
```diff
@@ -29,13 +30,11 @@ const data = [
...
 export default function SuccessStories() {
+  const { t } = useTranslation();
   return (
     <Container>
-      <h1 className="text-center title">Sucess Stories</h1>
-      <p className="text-center title">
-        We succeed to deliver the happiness to more than 1000 person, help us to
-        increase the number <br /> and participate in changing the world
-      </p>
+      <h1 className="text-center title">{t("successStories.header")}</h1>
```
Comment (Ammar-64) [inline src/components/SuccessStories/index.jsx:36]: There should be no h1s

File: src/components/PostList/index.jsx [hunk 3]
```diff
@@ -123,7 +124,7 @@ export default function PostList(props) {
           <select className="postFilter" id="city" onChange={handleCity}>
             <option value="all">{t("postPage.allLabel")}</option>
             {constantAllPosts.map((el) => {
-              return <option value={el.address[0]}>{el.address[0]}</option>;
+              return <option value={el.address}>{el.address}</option>;
```
Comment (Ammar-64) [inline src/components/PostList/index.jsx:127]: Are theses still working without the [0]

**Review by Ammar-64 — CHANGES_REQUESTED** [review]

File: src/components/PostList/index.jsx (hunk 3 above)
Comment (knargocer) [reply src/components/PostList/index.jsx:127]: Yes we changed the fireBase

**Review by Luey-A — APPROVED** [review]
LGTM

**Review by Ammar-64 — APPROVED** [review]
LGTM

**Pull request merged**